projects/plugins/beta/src/class-plugin.php

```diff
@@ -762,6 +773,21 @@ private function get_which_and_info( $source, $id ) {
 				$info->plugin_url = sprintf( 'https://github.com/%s', $this->mirror_repo() );
 				break;
```

Comment:
Again, we can get rid of the "master" case entirely, other than maybe keeping a `case 'master':` for back-compat if someone has a url bookmarked or something.

If we do keep a `case 'master':`, be sure to also assign `$source = 'trunk';` in the code.